docs/modules/tabs-v2/index.md

```diff
@@ -0,0 +1,548 @@
+
+![img](./_images/2.jpg)
+```javascript
+export default function SelectPreset( props ) {
+	const containerHelper = elementor.helpers.container,
+		onPresetSelected = ( preset, container ) => {
+			const options = {
+				createWrapper: false,
+			};
+
+			// Create new one by selected preset.
+			containerHelper.createContainerFromPreset( preset, container, options );
+		};
+
+	return (
+		<>
+			<div className="elementor-add-section-close">
+				<i onClick={() => props.setIsRenderPresets( false )} className="eicon-close" aria-hidden="true"/>
+				<span className="elementor-screen-only">{__( 'Close', 'elementor' )}</span>
+			</div>
+			<div className="e-view e-container-select-preset">
+				<div className="e-container-select-preset__title">{__( 'Select your Structure', 'elementor' )}</div>
+				<div className="e-container-select-preset__list">
+					{
+						elementor.presetsFactory.getContainerPresets().map( ( preset ) => (
+							<div onClick={() => onPresetSelected( preset, props.container )}
+								key={preset} className="e-container-preset" data-preset={preset}
+								dangerouslySetInnerHTML={{ __html: elementor.presetsFactory.generateContainerPreset( preset ) }}/>
+						) )
+					}
+				</div>
+			</div>
+		</>
+	);
+}
+
+SelectPreset.propTypes = {
+	container: PropTypes.object.isRequired,
+	setIsRenderPresets: PropTypes.func.isRequired,
+};
+```
+
+## Another useful feature that used in nested tabs is partial data render
+* Since the default render mechanism render all the children each render,
+	* There is issue with the nested-tabs, when you modify the `tab-title` for example, and the whole nested-tabs children hierarchy affected, it gets re-render which create huge performance impact
+	* The new nested infrastructure is allowing to avoid it, if you know the markup in advance using new feature called __partial render__:
+		* The feature will avoid the full re-render and select this exact node/markup and modify only it.
+		* You will be able to achieve that using the following markup in the element:
+			- For repeater items, example:
+      ```html
+        <element 
+            'data-binding-type': 'repeater-item',  // Type of binding (to know how to behave).
+            'data-binding-repeater-name': 'tabs',  // Repeater setting key that effect the binding.
+            'data-binding-setting': 'tab_title',   // The key in the repeater that effect the binding.
+            'data-binding-index': tabCount,        // Index is required for repeater items.
+        >
+        </element>
+      ```
+        - For simple settings items, example:
+	  ```html
+		 <element
+		 	'data-binding-type': 'content', 		 // Type of binding (to know how to behave).
+		 	'data-binding-setting': 'testimonial_content',   // Setting change to capture, the value will replace the data-binding.
+		 </element>
+	  ```
+      > Use it in the `_content_template()` method.
```

Comment:
Add an explanation about the global defaults in CSS variables

Reply:
Handled.